Backpack/Switch/Classes/BPKSwitch.h

```diff
@@ -26,6 +26,8 @@ NS_ASSUME_NONNULL_BEGIN
  */
 NS_SWIFT_NAME(Switch) IB_DESIGNABLE @interface BPKSwitch : UISwitch
 
+@property (nonatomic) UIColor *switchPrimaryColor UI_APPEARANCE_SELECTOR;
```

Comment:
I've thought about this some more and all it does is set `onTintColor` which is already marked `UI_APPEARANCE_SELECTOR` so maybe we could just piggyback off of that. If/when we support runtime themes we'll just need to have something that maps `switchPrimaryColor` -> `onTintColor`

Reply:
@k0nserv this is done now 😄